### ISSUE TITLE
I use pcap4j to analysis DNS,and found that StaticUdpPortPacketFactory.instanters.size()==0

### ISSUE THREAD

**Issue by LEO-1990** [kaitoy/pcap4j#202]
I use pcap4j to analysis DNS,and found that StaticUdpPortPacketFactory.instanters.size()==0.I get null in "PacketInstantiater instantiater = instantiaters.get(number)", the number is  (53, "Domain Name Server").I guess the problem is that instantiaters.put() in StaticUdpPortPacketFactory not work, but I don't know how to confirm and deal with it. Can you give me some suggestions?

**Comment by kaitoy** [comment]
Which version of Pcap4J do you use?
Newer versions' StaticUdpPortPacketFactory should have instanters with DNS support.

**Comment by LEO-1990** [comment]
Thanks for your reply.The version is pcap4j-1.7.5.And I have found the reason. I add code in pcap4j-packetfactory-static project,and when I package the project generated pcap4j-packetfactory-static:1.7.5,but the version of  pcap4j-packetfactory-static imported in the project pcap4j-core is 1.6.3. So the instanter I added doesn't work, and in pcap4j-packetfactory-static:1.6.3 there is no instanters added in StaticUdpPortPacketFactory.
